station.rb

```diff
@@ -0,0 +1,26 @@
+class Station
+  attr_reader :name
+
+  def initialize(name)
+    @name = name
+    @trains = []
```

Comment:
Trains array should contain trains, not the names only.

Reply:
same here